packages/api/bin/serveUtils.js

```diff
@@ -80,14 +80,13 @@ async function addGranules(granules) {
   const executionPgModel = new ExecutionPgModel();
   const es = await getESClientAndIndex();
   return await Promise.all(
-    granules.map(async (granule) => {
-      const newGranule = fakeGranuleFactoryV2(
-        {
-          ...granule,
-        }
-      );
-      await indexer.indexGranule(es.client, newGranule, es.index);
-      const dbRecord = await translateApiGranuleToPostgresGranule(newGranule, knex);
+    granules.map(async (g) => {
```

Comment:
I'm not sure this is right. The phase 3 branch likely removed the references to `dynamoRecord` and it looks like they were added back here.